### DISCUSSION TITLE
Problem with downloading

### DISCUSSION THREAD

**Discussion by Ku32** [KoboldAI/KoboldAI-Client#323] [General]
Command "git clone https://github.com/koboldai/koboldai-client" doesnt download all files

/home/kasm-user/koboldai-client/play.sh: line 3: ./install_requirements.sh: No such file or directory
/home/kasm-user/koboldai-client/play.sh: line 5: bin/micromamba: No such file or directory

Someone any ideas why this could happend?
I downloaded zip, unpacked it and it worked, but machine freezed in the end of instalation, and I restarted everything, deleted folder with kobold, unzipped the same file again, and again the same error showed up, it didnt had those files

**Comment by henk717** [comment]
Usually happens if wget or bzip2 are missing.

**Reply by Ku32** [reply]
problem solved, thanks
